src/schema/artist/index.js

```diff
@@ -32,15 +54,24 @@ import {
 } from "schema/auction_result"
 import ArtistArtworksFilters from "./artwork_filters"
 import filterArtworks from "schema/filter_artworks"
-import { connectionWithCursorInfo } from "schema/fields/pagination"
-import { RelatedArtists } from "./related"
-import { createPageCursors } from "schema/fields/pagination"
+import {
+  connectionWithCursorInfo
+} from "schema/fields/pagination"
+import {
+  RelatedArtists
+} from "./related"
+import {
+  createPageCursors
+} from "schema/fields/pagination"
```

Comment:
ugh -- i know this issue. For whatever reason I had this happen to me when I first started using VSCode.

@mzikherman - lets hop back on SH